.github/workflows/release.yml

```
@@ -41,20 +41,23 @@ jobs:
     strategy:
```

```suggestion
    name: Package Binary Release - ${{ matrix.platform }}
    strategy:
```

The default name is too long and contains all variables. We can shorten it.